tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -32,7 +30,6 @@
     {"endpoint": "facebook.com"},
     {"endpoint": "google.com"},
     {"endpoint": "s3.amazonaws.com"},
-    {"endpoint": "twitter.com"},
     {"endpoint": "wikipedia.org"},
```

Comment:
Rather than removing it completely, can we disable it only when S2N is built with Openssl-1.0.2? It seems like a waste to remove the working Openssl-1.1.1 test.